coverage.py

```diff
@@ -341,11 +341,45 @@ def work(self, identifier):
             return CoverageFailure(self, identifier, e, transient=True)
         return work
 
-    def set_metadata(self, identifier, metadata, 
+
+    def set_meta_circ_data(self, identifier, metadata, circulationdata, 
+        metadata_replacement_policy=None, 
+        circulationdata_replacement_policy=None, 
+    ):
+        """
+        Performs the function of the old set_metadata.  Finds or creates the Edition 
```

Comment:
This is on purpose.  I'm concerned there may be code not covered by the unit tests that would break.  In that case, it would be helpful to remember what the broken code was expecting.  I plan to remove these comments in a couple of months, once I know it's safe.